Kuroba/app/src/main/java/com/github/k1rakishou/chan/core/site/sites/dvach/DvachApiV2.kt

```diff
@@ -263,6 +266,21 @@ class DvachApiV2(
 
       builder.postImages(postImages, builder.postDescriptor)
 
+      if (threadPost.icon.isNotNullNorEmpty()) {
+        val document = Jsoup.parseBodyFragment(threadPost.icon)
+        val icons = document.body().select("img")
+        for (icon in icons) {
+          val imageUrl = icon?.attr("src")
+          val title = icon?.attr("title")
```

Comment:
I think you should add empty string + null checks for both of these, like:
```
icon?.attr("src")?.takeIf { attrValue -> attrValue.isNotNullOrEmpty } ?: continue 
icon?.attr("title")?.takeIf { attrValue -> attrValue.isNotNullOrEmpty } ?: continue 
```
Because if either of them is null or empty it may crash inside `endpoints.icon()`. 
This shouldn't normally happen but who knows so let's add these checks just in case.